### PR TITLE
Bump router golang version to 1.20.2

### PR DESCRIPTION
**Pull request by Tetrino**
Matches dependabot version upgrade.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-3.2.0
```
Comment (ChrisBAshton) [inline .ruby-version:1]: This looks like it slipped in. Probably should be a separate PR (especially as it's a downgrade)!

Comment (Tetrino) [reply .ruby-version:1]: Good catch. It shouldn't be needed at all. It'll sit here anyway until the k9s move is done.

**Review by chao-xian — APPROVED** [review]
